samples/file-encryptor/enclave/keys.cpp

```diff
@@ -290,9 +290,17 @@ int ecall_dispatcher::prepare_encryption_header(
     {
         TRACE_ENCLAVE("password_key");
         for (unsigned int i = 0; i < ENCRYPTION_KEY_SIZE_IN_BYTES; i++)
+        {
+            if (password_key[i] > UINT32_MAX)
```

Comment:
Discussed previously, but it is not possible for this condition to be true. What warning is this addressing?